wine.py

```diff
@@ -31,7 +31,7 @@ def wait_on(command):
     try:
         # Start the process in the background
         process = subprocess.Popen(command, stdout=subprocess.PIPE, stderr=subprocess.PIPE, text=True)
-        msg.cli_msg(f"Waiting on \"{' '.join(command)}\" to finish.", end='')
+        msg.cli_msg(f"Waiting on \"{' '.join(command)}\" to finish…")
         while process.poll() is None:
             msg.logos_progress()
             time.sleep(0.5)
```

Comment:
The reason for `end=''` here is so that the later `msg.logos_progress()` function can print a "progress" dot every 0.5 sec on the same line as "Waiting on ...". This change has the effect of printing the full message line plus a line return, then printing the dots (maybe only 5 at most) on the next line. Maybe that's what's intended?

Reply:
Ah, that explains it. I saw messages in the logs that had two periods which should only have one. I changed this line in order to circumvent "ugly" logs. Now I understand—yes, so the reason I saw two was the hard-coded one and an initial progress dot.

Hmm… I'm not sure the best method here, unless we pass along an additional var to determine whether or not to trigger progress dots… I can see the benefit of just keeping them as is but I can also see the benefit of having both options on hand, and it isn't like we're talking about a major issue… Hmm…

Reply:
What if I reverted this change and add a 1 second sleep before the timer kicks in?